aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -0,0 +1,478 @@
+}
+
+public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+{
+    app.UseAuthentication();
+    // Other app configuration removed.
+}
+```
+
+Options for certificate authentication include the ability to:
+
+* Accept self-signed certificates.
+* Check for certificate revocation.
+* Check that the proffered certificate has the right usage flags in it.
+
+A default user principal is constructed from the certificate properties. The user principal contains an event that enables supplementing or replacing the principal. For more information, see <xref:security/authentication/certauth>.
+
+[Windows Authentication](/windows-server/security/windows-authentication/windows-authentication-overview) has been extended onto Linux and macOS. In previous versions, Windows Authentication was limited to [IIS](xref:host-and-deploy/iis/index) and [HttpSys](xref:fundamentals/servers/httpsys). In ASP.NET Core 3.0, [Kestrel](xref:fundamentals/servers/kestrel) has the ability to use Negotiate, [Kerberos](/windows-server/security/kerberos/kerberos-authentication-overview), and [NTLM on Windows](/windows-server/security/kerberos/ntlm-overview), Linux, and macOS for Windows domain-joined hosts. Kestrel support of these authentication schemes is provided by the [Microsoft.AspNetCore.Authentication.Negotiate NuGet](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Negotiate) package. As with the other authentication services, configure authentication app wide, then configure the service:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddAuthentication(NegotiateDefaults.AuthenticationScheme)
+        .AddNegotiate();
+    // Other service configuration removed.
+}
+
+public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+{
+    app.UseAuthentication();
+    // Other app configuration removed.
+}
+```
+
+Host requirements:
+
+* Windows hosts must have [Service Principal Names](/windows/win32/ad/service-principal-names) (SPNs) added to the user account hosting the app.
+* Linux and macOS machines must be joined to the domain.
+
+  * SPNs must be created for the web process.
+  * [Keytab files](https://blogs.technet.microsoft.com/pie/2018/01/03/all-you-need-to-know-about-keytab-files/) must be generated and configured on the host machine.
+
+See <xref:security/authentication/windowsauth> for more information.
+
+## Template changes
+
+The web UI templates (Razor Pages, MVC with controller and views) have the following removed:
+
+* The cookie consent UI is no longer included. To enable the cookie consent feature in an ASP.NET Core 3.0 template generated app, see <xref:security/gdpr>.
+* Scripts and related static assets are now referenced as local files instead of using CDNs. For more information, see [this GitHub issue](https://github.com/aspnet/AspNetCore.Docs/issues/14350).
+
+The Angular template updated to use Angular 8.
+
+## Razor components
```

Comment:
```suggestion
## Blazor
```
I'd like to expand on this section and turn it into the Blazor section. The emphasis should be on writing client-side UI with .NET & C# instead of JavaScript. The should look something similar to what we have in the first section of the [Blazor overview](https://docs.microsoft.com/aspnet/core/blazor).

We should also have a nice list of the features support by the Blazor component model:

- Reusable UI components
- Client-side routing
- Component layouts
- Support for dependency injection
- Forms & validation
- Build component libraries with Razor Class Libraries
- JavaScript interop

We also need to be clear that support for Blazor Server apps is available with .NET Core 3.0, but support for Blazor WebAssembly apps is *still in preview*. We actually need to make this clear across the Blazor docs.

@guardrex Is this update something you can help out with?

Reply:
@danroth27 I added content as advised. You probably need to edit it.

Note that the title of what I added could be ...

\#\# Blazor and Razor components

... given what's in the section.